async-embedded/Cargo.toml

```diff
@@ -2,14 +2,25 @@
 authors = ["Jorge Aparicio <jorge.aparicio@ferrous-systems.com>"]
 edition = "2018"
 license = "MIT OR Apache-2.0"
-name = "async-cortex-m"
+name = "async-embedded"
 publish = false
 version = "0.0.0-alpha.0"
 
 [dependencies]
-cortex-m = "0.6.2"
-cortex-m-udf = { path = "../cortex-m-udf" }
 generic-array = "0.13.2"
 heapless = { git = "https://github.com/japaric/heapless", branch = "slab" }
 pin-utils = "0.1.0-alpha.4"
 typenum = "1.11.2"
+
+[target.'cfg(target_arch = "arm")'.dependencies]
+cortex-m = "0.6.2"
+cortex-m-udf = { path = "../cortex-m-udf" }
+
+[target.'cfg(any(target_arch = "riscv32", target_arch = "riscv64"))'.dependencies]
+riscv = "0.5.6"
+
+[features]
+default = ["riscv-wait-nop"]
+riscv-wait-nop = []
+riscv-wait-wfi-single-hart = []
+wait-extern = []
```

Comment:
```suggestion
riscv-wait-extern = []
```

Reply:
Apart from this small issue everything else looks good to me

Reply:
Nice catch.

I wanted to keep the wait-extern ISA-agnostic, since other ISA's might need a similar backup soltion. But let's keep this PR focused on RISC-V.